src/js/exporters.js

```diff
@@ -272,6 +276,57 @@ function exportKontaktScript() {
   return true
 }
 
+function exportImageLinePitchMap(range) {
+  if (exportError()) {
+    return
+  }
+  const NB_NOTES = 121 // IL products can only retune from C0 to C10
+  const HEADER_BYTES = Uint8Array.from([3, 0, 0, 0, 3, 0, 0, 0, NB_NOTES, 0, 0, 0])
+  const ENDING_BYTES = Uint8Array.from([0, 0, 0, 0, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255])
+
+  const tuningTable = model.get('tuning table')
+  const baseFreqOffset = Math.log2(tuningTable.baseFrequency / 440) // in number of octaves
+
+  // construct point data
+  let points = new ArrayBuffer(121 * 24)
+  let pointsDoubles = new Float64Array(points)
+  let pointsUint32 = new Uint32Array(points)
+  for (let i = 0; i < NB_NOTES; i++) {
+    const edo12cents = (i - 69) * 100
+    const offset = tuningTable.cents[i] - edo12cents
+    const normalizedOffset = ((offset / 1200 + baseFreqOffset) / range) * 0.5 + 0.5
+    const yCoord = Math.max(0.0, Math.min(1.0, normalizedOffset))
+    pointsDoubles[i * 3 + 1] = yCoord
+    if (i !== 0) { // no x offset and no curve data for first point
+      pointsDoubles[i * 3] = 0.0083333337679505 // constant x offset from previous point
+      pointsUint32[i * 6 + 4] = 0        // |
+      pointsUint32[i * 6 + 5] = 33554432 // | no curve
```

Comment:
This raises the same question as before: where is this number coming from and can this be calculated? Please also move this number out to a constant to give it some context.